app/js/trouble.js

```diff
@@ -53,15 +55,15 @@
 			$dropdown.data("status", 'closed');
 			$dropdown.removeClass('gbl_dropdown_active');
 			Array.prototype.foreach = function(cb) {
-				for (i = 0; i < this.length; ++i) {
+				for (var i = 0; i < this.length; ++i) {

```

Comment:
Ideally the var declaration appears at the top of the function, but this works for clearing the lint error.